pkg/azuredisk/nodeserver_v2.go

```diff
@@ -519,3 +549,89 @@ func (d *DriverV2) ensureMountPoint(target string) (bool, error) {
 	klog.Warning("ensureMountPoint method is deprecated.")
 	return d.nodeProvisioner.EnsureMountPointReady(target)
 }
+
+func (d *DriverV2) recoverMount(diskURI string) {
+	// if mount recovery is necessary (if time.Now is past the set nextRetry), detach volume so that volume can be reattached via reconciliation
+	if d.isMountRecoveryRequired(diskURI) {
```

Comment:
np: Consider `shouldStartRecovery` since that is the question we're really asking here.